src/App.tsx

```diff
@@ -1,7 +1,40 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string,
+  initial: boolean,
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+    initial: true,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeyUp);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeyUp);
+  }
+
+  handleKeyUp = (event: KeyboardEvent) => {
+    this.setState({
+      pressedKey: event.key,
+      initial: false,
+    });
+  };
+
+  render() {
+    const { pressedKey, initial } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__message">
+          {initial ? 'Nothing was pressed yet' : `The last pressed key is [${pressedKey}]`}
```

Comment:
you can use just pressedKey value:
```suggestion
          {!pressedKey ? 'Nothing was pressed yet' : `The last pressed key is [${pressedKey}]`}
```

Empty string is false

Reply:
@AlinaMatuschak Great advice, I take my hat off to you 👍